### DISCUSSION TITLE
Microsoft Sharepoint/Stream Video Download Capability

### DISCUSSION THREAD

**Discussion by stefanjt** [aclap-dev/video-downloadhelper#579] [Help and Q&A]
### Your question

Hi. Can you please add this feature? I'm using v. 9.0.1.13 on Chrome 124.0.6367.210 on Windows 11 with CoApp v. 2.0.19.0 and I'm getting the below error message when trying to download.from Sharepoint:

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://koenigsolutionsltd-my.sharepoint.com/personal/krishna_dwivedi_koenig-solutions_com/_layouts/15/stream.aspx?id=%2Fpersonal%2Fkrishna%5Fdwivedi%5Fkoenig%2Dsolutions%5Fcom%2FDocuments%2FRecordings%2FILO%5F191936%20for%20Course%5F%20AI%2D900T00%5F%20Microsoft%20Azure%20AI%20Fundamentals%2D20240522%5F192957%2DMeeting%20Recording%2Emp4&ga=1&referrer=StreamWebApp%2EWeb&referrerScenario=AddressBarCopied%2Eview%2E881db0a3%2Da2af%2D4a6a%2D973c%2Da83faae1b134",
  "error": {
    "details": "SideDownload error: : [dash @ 000002ce7a1b9800] Error when loading first fragment of playlist\r\n[in#0 @ 000002ce7a1b9640] Error opening input: Invalid data found when processing input\r\nError opening input file https://southindia1-mediap.svc.ms/transform/videomanifest?provider=spo&inputFormat=mp4&cs=fFNQTw&correlationId=78832ba1-208e-3000-60a2-a11c95cc6554&docid=https%3A%2F%2Fkoenigsolutionsltd-my.sharepoint.com%2Fpersonal%2Fkrishna_dwivedi_koenig-solutions_com%2F_api%2Fv2.0%2Fdrives%2Fb%21-e2aL_xsFECULFKOMi2f6VxYAJelam9IsDu9vtuW5zrCj7LzFfwoT5B59emU-7-C%2Fitems%2F01PLYENLR5TOVMMHNFK5CLMF42YJDC7GLT%3Ftempauth%3DeyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiIwMDAwMDAwMy0wMDAwLTBmZjEtY2UwMC0wMDAwMDAwMDAwMDAva29lbmlnc29sdXRpb25zbHRkLW15LnNoYXJlcG9pbnQuY29tQDk4ZGViMTRhLThmMmYtNDhiMi04MDdmLThhOTdjOTZhMGNhMyIsImNhY2hla2V5IjoiMGguZnxtZW1iZXJzaGlwfHVybiUzYXNwbyUzYWFub24jZTI2MmExNjdkYmI3ZGIzYTBkZDMzNDUwM2Y1Y2Q0Mjg3YWIyODM3YTU4OGVmZjkzMTM2ZjE3YWI3NDIzNTEyOSIsImNpZCI6Im9TdURlSTRnQURCZ29xRWNsY3hsVkE9PSIsImVuZHBvaW50dXJsIjoiZW11cDBMT2l4SXpna3NiczZMTUxYNjNxU2FUeEJ5dHRCeGxpby93b1pWOD0iLCJlbmRwb2ludHVybExlbmd0aCI6IjIxNSIsImV4cCI6IjE3MTY0OTkyNzAiLCJpcGFkZHIiOiIxNTEuMjUxLjYzLjE1MyIsImlzbG9vcGJhY2siOiJUcnVlIiwiaXNzIjoiMDAwMDAwMDMtMDAwMC0wZmYxLWNlMDAtMDAwMDAwMDAwMDAwIiwiaXN1c2VyIjoidHJ1ZSIsIm5hbWVpZCI6IjAjLmZ8bWVtYmVyc2hpcHx1cm4lM2FzcG8lM2Fhbm9uI2UyNjJhMTY3ZGJiN2RiM2EwZGQzMzQ1MDNmNWNkNDI4N2FiMjgzN2E1ODhlZmY5MzEzNmYxN2FiNzQyMzUxMjkiLCJuYmYiOiIxNzE2NDc3NjcwIiwibmlpIjoibWljcm9zb2Z0LnNoYXJlcG9pbnQiLCJzaGFyaW5naWQiOiJ0V29KSEVjYmYwU0oxaGV3djIzbXpRIiwic2l0ZWlkIjoiTW1ZNVlXVmtaamt0Tm1ObVl5MDBNREUwTFRrME1tTXROVEk0WlRNeU1tUTVabVU1IiwidHQiOiIwIiwidmVyIjoiaGFzaGVkcHJvb2Z0b2tlbiJ9.pA61cPS1Ne0g68l1ZE3pecelEqumqlGVStdlrzHCdnw%26version%3DPublished&psi=881db0a3-a2af-4a6a-973c-a83faae1b134&enhanceAudio=true&ccat=1&cTag=%22c%3A%7BC6AA9B3D-A51D-4457-B617-9AC2462F9973%7D%2C2%22&action=Access&part=index&format=dash&altManifestMetadata=eyJBdWRpb0NvZGVjSW5mbyI6ImFhYyIsIkF1ZGlvRnJhbWVTaXplIjoxMDI0LCJBdWRpb1NhbXBsaW5nUmF0ZSI6MTYwMDAsIkF1ZGlvU3RhcnRUaW1lSW4xMDBOYW5vIjowLCJBdWRpb1N0cmVhbUR1cmF0aW9uMTAwTmFubyI6MTQ0MDAwNjQwMDAwLCJCaXRyYXRlIjo0NjI2OTUsIkNUYWciOm51bGwsIkNvbnRhaW5lclN0YXJ0VGltZUluMTAwTmFubyI6MCwiQ29udGFpbnNBdWRpbyI6dHJ1ZSwiQ29udGFpbnNWaWRlbyI6dHJ1ZSwiQ29udGVudEF3YXJlQml0cmF0ZSI6bnVsbCwiQ29udGVudENhdGVnb3J5IjoidGVhbXNfbWVldGluZyIsIkNyZWF0aW9uVW5peFRpbWVzdGFtcFNlY29uZHMiOjE3MTY0MDEwMzMsIkNyeXB0b2dyYXBoaWNEYXRhQ3JlYXRlZERhdGVVbml4IjoxNzE2NDAxMDM0LCJDcnlwdG9ncmFwaGljRGF0YUlWIjoiMHhGNUIyQzU5RTlGMjdCQTdGM0Y3QUZERTAyRjRFMzYwRiIsIkNyeXB0b2dyYXBoaWNEYXRhS2lkIjoiMmQ1OTc2MTljNzQuODgwNDYwLjU2ODcxIiwiRHVyYXRpb24xMDBOYW5vIjoxNDQwMDA2NDAwMDAsIkZyYW1lU2l6ZSI6eyJoZWlnaHQiOjEwODAsIndpZHRoIjoxOTIwfSwiR29wU2l6ZSI6NjAwMDAwMDAsIkgyNjRMZXZlbCI6NDAsIkhhc0JGcmFtZXMiOmZhbHNlLCJIZWFkZXJPZmZzZXRJbkJ5dGVzIjoyNCwiSGVhZGVyU2l6ZSI6NDA2NTYyOCwiTWV0YWRhdGFTb3VyY2VJbmZvcm1hdGlvbiI6eyJBdWRpb1N0cmVhbUR1cmF0aW9uU291cmNlIjoxLCJBdWRpb1N0cmVhbVJlYWRCdWZmZXJTaXplSW5CeXRlcyI6MTYzODQsIkNvbnRhaW5lckR1cmF0aW9uU291cmNlIjoxLCJWaWRlb0JpdHJhdGVTb3VyY2UiOjEsIlZpZGVvU3RyZWFtRHVyYXRpb25Tb3VyY2UiOjEsIlZpZGVvU3RyZWFtUmVhZEJ1ZmZlclNpemVJbkJ5dGVzIjoxNjM4NH0sIlBhY2tlZEZyYW1lUmF0ZSI6eyJIaWdoIjoyODgwMDEyNSwiTG93IjoxODAwMDA4fSwiUHJlbWl1bVZpZGVvUHJldHJhbnNjb2RlRGF0YSI6bnVsbCwiUmVhbEJhc2VGcmFtZVJhdGUiOnsiSGlnaCI6MTYsIkxvdyI6MX0sIlVwZGF0ZWFibGVBdWRpb1RyYWNrcyI6W10sIlZlcnNpb24iOnsiX0J1aWxkIjotMSwiX01ham9yIjo2LCJfTWlub3IiOjAsIl9SZXZpc2lvbiI6LTF9LCJWaWRlb0NvZGVjSW5mbyI6ImgyNjQiLCJWaWRlb1ByZXRyYW5zY29kZURhdGEiOnsiQXVkaW9RdWFsaXR5SWRzIjoiYXVkY29weSIsIlNlZ21lbnRDb3VudCI6MiwiVmlkZW9RdWFsaXR5SWRzIjoidmNvcHkifSwiVmlkZW9TdGFydFRpbWVJbjEwME5hbm8iOjAsIlZpZGVvU3RyZWFtRHVyYXRpb24xMDBOYW5vIjoxNDQwMDA2NDAwMDB9&altTranscode=1&useScf=true&pretranscode=0&transcodeahead=0&enableCdn=1&hybridPlayback=false&pn=hostApp-OnePlayer-Web.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_5871642830209076",
    "error": "coapp_failure",
    "id": "coapp_failure_8d40f2cf-1dc9-429b-9993-07f0a58a5fb5"
  }
}

### Addon & CoApp version and details

Chrome extension: v. 9.0.1.13
Chrome: 124.0.6367.210
OS: 
Windows 11 Enterprise build 22631.3593
CoApp: v. 2.0.19.0

**Comment by Richardvi** [comment]
I really need this to. Microsoft Teams meetings are being recorded by person A. Person A shares this recording for viewing, but downloading the recording is not possible. I need to download these recording. Video DownloadHelper just says that this is not supported.

**Comment by jcfrog** [comment]
Hi,
It would help to have a public link for a media but I guess it might not be possible.

**Comment by Richardvi** [comment]
Hmm... yes that would be difficult I guess. MS Teams records meetings and the recordings go to OneDrive. You can also choose to view those recordings in Microsoft Stream: https://www.microsoft.com/nl-nl/microsoft-365/microsoft-stream

You can try for yourself if you can find such a recording, but if you send me your email address I can try to give you access to one of my recordings. I do have to give you an account in my MS tenant though. We can not share recordings outside of our tenant.

**Reply by jcfrog** [reply]
The site has been added to the to do list for the dev.
